cmd/user-manager/manager/manager.go

```diff
@@ -0,0 +1,157 @@
+package manager
+
+import (
+	"database/sql"
+	"log"
+	"os"
+
+	_ "github.com/go-sql-driver/mysql"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v2"
+)
+
+type Manager struct {
+	db         *sql.DB
+	secretPath string
+	Users      []User
+}
+
+type UsersSecret struct {
+	StringData map[string]string `yaml:"stringData"`
+}
+
+type Data struct {
+	Users []User `yaml:"users"`
+}
+
+type User struct {
+	Name   string   `yaml:"username"`
+	Pass   string   `yaml:"password"`
+	Tables []Table  `yaml:"tables"`
+	Hosts  []string `yaml:"hosts"`
+}
+
+type Table struct {
+	Name       string `yaml:"name"`
+	Privileges string `yaml:"privileges"`
+}
+
+func New(hosts []string, rootPass, secretPath string) (Manager, error) {
+	var um Manager
+	var err error
+	for _, host := range hosts {
+		mysqlDB, err := sql.Open("mysql", "root:"+rootPass+"@tcp("+host+")/?interpolateParams=true")
+		if err != nil {
+			log.Println(errors.Wrap(err, "create db connection"))
+			continue
+		}
+		um.db = mysqlDB
+		log.Println("using  host: " + host)
+		break
+	}
+	if um.db == nil {
+		return um, errors.Wrap(err, "cannot connect to any host")
+	}
+	um.secretPath = "./data/secret.yaml"
+	if len(secretPath) > 0 {
+		um.secretPath = secretPath
+	}
+	return um, nil
+}
+
+func (u *Manager) GetUsers() error {
+	file, err := os.Open(u.secretPath)
+	if err != nil {
+		return errors.Wrap(err, "open secret file")
+	}
+	var data Data
+	err = yaml.NewDecoder(file).Decode(&data)
+	if err != nil {
+		return errors.Wrap(err, "unmarshal secret")
+	}
+	u.Users = data.Users
+
+	return nil
+}
+
+func (u *Manager) ManageUsers() error {
+	defer u.db.Close()
+	tx, err := u.db.Begin()
+	if err != nil {
+		return errors.Wrap(err, "begin transaction")
+	}
+	for _, user := range u.Users {
```

Comment:
we should support way how to remove users :(

Reply:
Added https://github.com/percona/percona-xtradb-cluster-operator/pull/338/commits/20c19419e5df24e56e6f8461540c70aa7480932e